src/answers-umd.js

```diff
@@ -282,7 +292,9 @@ class Answers {
     });
 
     if (parsedConfig.onStateChange && typeof parsedConfig.onStateChange === 'function') {
-      parsedConfig.onStateChange(persistentStorage.getAll(), window.location.search.substr(1));
+      parsedConfig.onStateChange(
+        Object.fromEntries(storage.getAll()),
```

Comment:
nit: This is out of scope of this PR, but what are your thoughts on changing storage.getAll() to return the object so that we don't have to call Object.fromEntries? We could also have a storage.getAllAsMap() if we want the Map object specifically

Reply:
hmm I'm not really sure on the pros or cons, right now Storage uses a Map internally not an object so it seems slightly simpler this way but I don't really mind which way